hack/validation/resources.sh

```diff
@@ -1,7 +1,4 @@
-# Adding validation to both v1 and v1beta1 APIs
-# Version = 0 // v1 API 
-# Version = 1 // v1beta1 API
-for Version in  $(seq 0 1); do
-    # Adding validation for nodepool.spec.template.spec.resources 
-    yqVersion="$Version" yq eval ".spec.versions[env(yqVersion)].schema.openAPIV3Schema.properties.spec.properties.template.properties.spec.properties.resources.maxProperties = 0" -i pkg/apis/crds/karpenter.sh_nodepools.yaml 
-done
\ No newline at end of file
+# Adding validation v1beta1 NodePool API
+
+# Adding validation for nodepool.spec.template.spec.resources 
+yqVersion="$Version" yq eval ".spec.versions[1].schema.openAPIV3Schema.properties.spec.properties.template.properties.spec.properties.resources.maxProperties = 0" -i pkg/apis/crds/karpenter.sh_nodepools.yaml 
```

Comment:
I don't see the yqVersion envVar referenced in this line. Am I missing something?

Reply:
This was included to indicate the version. Removed it now since we only need to reason about v1beta1